spec/lib/abstractivator/value_map_spec.rb

```diff
@@ -22,7 +22,7 @@
   end
 
   it 'maps nil' do
-    expect(nil.value_map(&method(:square))).to be nil
+    expect(nil.value_map(&:to_s)).to eql ''
```

Comment:
How does 'Nil' still have `value_map` if you removed it from the change in `value_map.rb`?

Reply:
```
irb> NilClass.superclass
=> Object
```

Reply:
Ah, of course!